lib/components/CentralDevice.jsx

```diff
@@ -166,11 +166,11 @@ class CentralDevice extends React.PureComponent {
 
             if (onToggleAutoConnUpdate) {
                 items.push(<Dropdown.Divider key="dividerConnUpdate" />);
-                items.push(<Dropdown.Header key="connUpdateHeader">Connection update</Dropdown.Header>);
+                items.push(<Dropdown.Header key="connUpdateHeader">Connection, phy and data length update</Dropdown.Header>);
                 items.push(
                     <Dropdown.Item
                         key="autoConnUpdate"
-                        title="Automatically accept connection update requests"
+                        title="Automatically accept connection, phy and data length update requests"
```

Comment:
Here the data length is referred to but not MTU, whereas other places it is the opposite. Should be consistent if possible.

Reply:
The auto reply to MTU requests in pc-ble-driver-js was not changed, and for a 51 device you can change only the MTU - not the data length, so this will be a bit misleading. Should I just change the title here or should I work further on then driver to make the MTU request pop up?

Reply:
Bear in mind that when changing data length and MTU for 52 together, one has to complete before the other can be set. On the receiving side it would look quite odd if the user needs to accept 2 events after each other.

Reply:
Ok, I though the auto response was removed in ble-driver-js. I think that the two requests should be treated equally. Either auto reply to both in ble-driver-js or forward both requests to the app.